tests/cypress/e2e/features/consensus.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Basic manipulations with consensus job replicas', () => {
+    describe('Consensus job creation', () => {
+        // const jobType = 'consensus_replica';
+        const maxReplicas = 10;
+        const taskName = 'Test consensus';
+        const labelName = 'test';
+        const serverFiles = ['archive.zip'];
+        const replicas = 3;
+        before(() => {
```

Comment:
```suggestion
        const replicas = 3;
        
        before(() => {
```
Please, add newlines between functions here and all over the file

Reply:
Ok, adding newlines